hardware/benchmark/disk.py

```diff
@@ -41,6 +43,8 @@ def is_booted_storage_device(disk):
     grep_cmd = subprocess.Popen(cmdline,
                                 shell=True, stdout=subprocess.PIPE)
     for booted_disk in grep_cmd.stdout:
+        if not isinstance(booted_disk, six.text_type):
+            booted_disk = six.text_type(booted_disk)
```

Comment:
This is probably not doing what you think it's doing. In python 3 str(bytes) is a representation of a bytes object:
```
>>> str(b'')
"b''"
>>> len(str(b''))
3
```

Reply:
mmm to be honest I don't remember why I was doing that
anyway! thanks, updated